front/src/applications/carto/Map.tsx

```diff
@@ -138,6 +141,8 @@ function Map() {
           </>
         )}
 
+        <VirtualLayers />
```

Comment:
It’s already there (line 120 ;))

Reply:
Yes ! In fact, it comes twice when I rebased with your pr ;) I'll keep the first occurence.